README.md

```diff
@@ -177,6 +177,33 @@ node example.js --foo.bar
 { _: [], "foo.bar": true }
 ```
 
+_Note:_ This setting does not apply to config files (any more). See `config-dot-notation`.
```

Comment:
Perhaps:

`_Note: as of yargs-parser@14.0.0, `dot-notation` has been split into `dot-notation` and `config-dot-notation`_`

Reply:
Ok, I was expecting this :)